src/torrent/utils/extents.h

```diff
@@ -17,6 +17,8 @@ class extents {
 
   extents();
   ~extents();
+  extents(const extents&) = default;
```

Comment:
actually this is hillarious. This class' special member functions should probably all be removed.